src/ui/server/constants/field-ids/insurance/policy/index.ts

```diff
@@ -49,7 +49,12 @@ export const POLICY = {
   },
   NEED_PRE_CREDIT_PERIOD: 'needPreCreditPeriodCover',
   CREDIT_PERIOD_WITH_BUYER: 'creditPeriodWithBuyer',
-  NEED_ANOTHER_COMPANY_TO_BE_INSURED: 'needAnotherCompanyToBeInsured',
+  REQUEST_JOINTLY_INSURED_PARTY: 'requestJointlyInsuredParty',
+  REQUESTED_JOINTLY_INSURED_PARTY: {
+    COMPANY_NAME: 'companyName',
```

Comment:
or maybe these should be shared fieldIds?

Reply:
Great idea RE both previous comments, just had a look. Can we tackle this a tech improvement? We can have shared field IDs, but would need to restructure all instances of Companies House to use the same IDs. Don't want to bloat this PR and keen to unblock other tickets

Reply:
Made a note to create a ticket for this